plugins/katello/3.7/release_notes/release_notes.md

```diff
@@ -4,71 +4,36 @@ title: Release Notes
 version: 3.7
 ---
 
-# Katello 3.6 (Imperial IPA) Release Notes
+# Katello 3.7 (Blonde Ale) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.6/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.7/CHANGELOG.md).
 
-## Warning
+## Known RC1 Issues
+* Upgrades from 3.6 to 3.7 are not fully functional due to a failing database migration
+* Client tool repos do not all have the proper gofer versions resulting in no connectivity between Katello and affected clients
 
-A disclaimer, we are aware of a bug in 3.6 where you cannot delete an organization that has a host. Before deleting an organization manually delete any host attached to that organization. [Issue on Redmine](http://projects.theforeman.org/issues/23262)
 
 ## Features
 
-### Content View Optimizations
 
-Performance improvements across the board have brought up to a 50% improvement in Content View Publishes, 95% improvement in Promotions, and a 95% improvement in Composite Content View publishes.
+### New Red Hat Repositories Page
+Completely redesigned with an eye toward usability and functionality.
 
-## Deprecation Warnings
+### New Upstream Subscriptions API
+For users with Red Hat content we now provide an API for managing subscriptions against existing manifests. Check the API documentation for details.
 
-#### File Repository Paths on Upgrade
+### Content Views
+Composite content views will now be auto-published when one of its components is changed.
 
-Up until now file repositories were not able to be synced out to a proxy due to the URL path that the published file repositories were generated at. Katello 3.6 modifies the published path to mirror paths similar to how yum and puppet modules are published. This allows proper HTTPS serving of files repositories and syncing them out to proxy along with a more obvious and readable path. The downside of this needed change is that all file repositories are republished to a new path rendering all previous paths incorrect for any changes to the file repositories from the point of upgrade forward. The original file repository paths are left in tact, snapshotted at the point of the upgrade. This will allow a small window to update any infrastructure that relies on those file paths.
+### Client Tooling
+Zypper and DNF plugins have been added to support errata applicability similar to Yum clients
```

Comment:
Are we also delivering subscription-manager for suse clients as part of the foreman/katello repos?

Reply:
I think we rely on base repos or Red Hat repos for that as I don't see subscription-manager in our client repos. @ehelms can you validate that theory?